src/api.v2/helpers/s3/patchCellSetsObject.js

```diff
@@ -6,13 +6,28 @@ const putObject = require('./putObject');
 
 const validateRequest = require('../../../utils/schema-validator');
 
+function removeElementsWithTypeCLM(data) {
+  // Filter out cell-level metadata cell sets
+  return data.filter((item) => item.type !== 'CLM' && item.type !== 'CLMPerSample');
+}
+
+const containsTypeCLM = (data) => data.some((item) => {
+  const appendObject = item.$append;
```

Comment:
Can you just check if `item` is array? Why this way?

Reply:
I think it's more accurate to check if any is CLM in case we add other cell sets (which we want to do in the future)